pkgs/by-name/do/dooit/package.nix

```
@@ -5,53 +5,62 @@
   python311,
   testers,
   nix-update-script,
+  extraPackages ? [ ],
 }:
 let
   python3 = python311;
+  username = "dooit-org";
+  repo = "dooit";
+  ver = "v3.0.2";
 in
 python3.pkgs.buildPythonApplication rec {
-  pname = "dooit";
-  version = "2.2.0";
+  pname = repo;
```

```suggestion
  pname = "dooit";
```
https://github.com/nix-community/nixpkgs-lint/issues/21